assets/js/components/optin.js

```diff
@@ -52,9 +53,27 @@ class Optin extends Component {
 			method: 'POST',
 		} )
 			.then( () => {
+				if ( !! checked && ! this.state.scriptOnPage ) {
+					const { jQuery } = window;
+
+					jQuery( 'body' ).append( `
+						<script async src="https://www.googletagmanager.com/gtag/js?id=${ googlesitekit.admin.trackingID }"></script>
+					` );
+					jQuery( 'body' ).append( `
```

Comment:
You can chain the second `.append()` right off the first; no need to re-query the element 👍

Reply:
ahaha, right. It's been ages since I've used jQuery 😓